webpack.common.js

```diff
@@ -0,0 +1,26 @@
+const path = require('path');
+
+module.exports = {
+  entry: './src/index.js',
+  output: {
+    path: path.resolve(__dirname, 'dist'),
+    libraryTarget: 'umd',
+    library: 'LinxBanner',
+    globalObject: 'typeof self !== \'undefined\' ? self : this',
+  },
+  module: {
+    rules: [
+      {
+        test: /\.js$/,
+        exclude: /node_modules/,
+        loader: 'babel-loader',
+      },
+      {
+        enforce: 'pre',
```

Comment:
what is this?

Reply:
[This rule](https://webpack.js.org/configuration/module/#rule-enforce) tell which block should come first. In this case, it tells eslint should be executed before code generation by babel.